timely/examples/exchange.rs

```diff
@@ -15,7 +16,7 @@ fn main() {
         let probe = worker.dataflow(|scope|
             scope
                 .input_from(&mut input)
-                .exchange(|&x| x as u64)
+                .exchange_pact(LazyExchange::new(|&x| x as u64))
```

Comment:
Nit, but is this an important change, or just part of your testing?

Reply:
Reverted the changes here.